### DISCUSSION TITLE
Support bumpVersion for all languages

### DISCUSSION THREAD

**Discussion by gregoryfischer** [renovatebot/renovate#14186] [Suggest an Idea]
### What would you like Renovate to be able to do?

Support bumpVersion for all languages. 

I find that even minor version bumps can break a maven application and patching is only safe to perform in an automated fashion making "bumpVersion": "patch" essential for maven.

Ideally, I would like to see all version refinement configs to be applied across all languages and packages.

### If you have any ideas on how this should be implemented, please tell us here.

none

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
I think you misunderstand what the config option bumpVersion does.

Instead please describe in more detail what you mean.

**Comment by gregoryfischer** [comment]
@rarkins - my apologies if I misunderstood the bumpVersion configuration option. What I am interested in being able to restrict maven version updates to only those of type "patch" where both "minor" and "major" version updates would be ignored.

**Reply by rarkins** [reply]
If you search these discussions for mentions of separateMinorPatch and matchUpdateTypes then you should see examples of how to configure it
